tests/templates/y_cable_simulator_client.j2

```diff
@@ -172,7 +180,7 @@ def _post(physical_port, data):
         return False
     return True
 
-def _get(physical_port):
+def _get(physical_port, action=None):
```

Comment:
Why add this `action` argument? I don't see that it is used.

Reply:
I think it's used for flap counter in line 335